flink-runtime/src/main/java/org/apache/flink/runtime/state/filesystem/FsCheckpointStorage.java

```diff
@@ -78,11 +81,6 @@ public FsCheckpointStorage(
 		this.sharedStateDirectory = new Path(checkpointsDirectory, CHECKPOINT_SHARED_STATE_DIR);
 		this.taskOwnedStateDirectory = new Path(checkpointsDirectory, CHECKPOINT_TASK_OWNED_STATE_DIR);
 		this.fileSizeThreshold = fileSizeThreshold;
```

Comment:
You could remove the `throws IOException` from this constructor. It is no longer needed.

Reply:
I noticed that there are more methods here with unnecessary `throws IOException` in this class. If you want you could remove them in a hotfix commit and make that part of the PR.